cmd/webhook/main.go

```diff
@@ -191,8 +188,6 @@ func newConversionController(ctx context.Context, cmw configmap.Watcher, brokers
 		eventsv1alpha1_    = eventsv1alpha1.SchemeGroupVersion.Version
 		eventsv1beta1_     = eventsv1beta1.SchemeGroupVersion.Version
 		eventsv1_          = eventsv1.SchemeGroupVersion.Version
-		messagingv1alpha1_ = messagingv1alpha1.SchemeGroupVersion.Version
-		messagingv1beta1_  = messagingv1beta1.SchemeGroupVersion.Version
```

Comment:
@capri-xiyue sorry I don't know why your comment is gone after my force push. 

> @capri-xiyue why do we need to remove v1beta1?

this variable is only used below in the conversion map, since we're removing v1alpha1, there's no need to convert.

Reply:
I believe it is also used in validation and defaults webhook. You need to search the usage of `var types`.

Reply:
see comment above.